CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.10.0 / 2019-05-22
+
+* [FEATURE] Template expansion: Make external labels available as `$externalLabels` in alert and console template expansion. #5463
+* [FEATURE] Web UI: Sort alerts by group name. #5448
+* [FEATURE] TSDB: Add `prometheus_tsdb_wal_segment_current` metric for the WAL segment index that TSDB is currently writing to. tsdb#601
+* [FEATURE] Scrape: Add `scrape_series_added` per-scrape metric. #5546
+* [FEATURE] Log filesystem type on startup. #5558
```

Comment:
ENHANCEMENT, and I'm not sure I'd list it as it's mostly for our own support purposes.

Reply:
I'd still list it. Happy to demote it to ENHANCEMENT.